odoo/models.py

```diff
@@ -1468,6 +1468,14 @@ def user_has_groups(self, groups):
 
         return not has_groups
 
+    @api.model
+    def check_domain(self, domain):
+        try:
+            return self.search_count(domain)
+        except Exception as e:
+            e.sentry_ignored = True
+            raise
```

Comment:
It's interesting idea. 
I'd use another either use another name (`search_count_silent` ?) or keep the name, but return dictionary instead (e.g. `{recordCount: 0, isValid: true/false }`).